CentOS build action sporadically running OOM

The github build action for CentOS is inconsistently running out of memory during some sort of `packman` installation stage. 
Here's an example output from [this failed run](https://github.com/CesiumGS/cesium-omniverse/actions/runs/5211978370/jobs/9404965628)

```
Package successfully installed to /var/tmp/packman/chk/nv-usd/20.08.nv.1.2.2568.0bec9a14-linux64_py37-centos_debug-main
Package 'rtx_plugins' at version '104.2+release.104.2.271-890c40c7-release.890c40c7.tc.linux-x86_64' is missing from local storage.
Downloading from https://d4i3qtqj3r0z5.cloudfront.net/rtx_plugins%40104.2%2Brelease.104.2.271-890c40c7-release.890c40c7.tc.linux-x86_64.7z (1.29 GiB)
No continuous progress report because this is not a proper terminal. Be patient ...
100% (speed 168 MiB/s)
Total of 7.88 seconds
No continuous progress report because this is not a proper terminal. Patience is a virtue ...
Traceback (most recent call last):
  File "/github/workspace/extern/nvidia/tools/repoman/build.py", line 35, in <module>
    run_command()
  File "/github/workspace/extern/nvidia/tools/repoman/build.py", line 31, in run_command
    packmanapi.pull(os.path.join(deps_folder, deps_target), platform=options.platform_target)
  File "/var/tmp/packman/packman-common/6.45.1/packman/utils.py", line 551, in wrapper_message_block
    return func(*args, **kwargs)
  File "/var/tmp/packman/packman-common/6.45.1/packman/main.py", line 1120, in pull
    project_path, platform, postscript, remotes, include_tags, exclude_tags, tokens
  File "/var/tmp/packman/packman-common/6.45.1/packman/main.py", line 1177, in pull_with_variable_file
    args=args,
  File "/var/tmp/packman/packman-common/6.45.1/packman/main.py", line 1338, in pull_dependencies
    packager.install_package(target_filename, install_path)
  File "/var/tmp/packman/packman-common/6.45.1/packman/packager.py", line 320, in install_package
    archive7z.extract_archive_to_folder(package_path, output_folder)
  File "/var/tmp/packman/packman-common/6.45.1/packman/archive7z.py", line 254, in extract_archive_to_folder
    archive.raise_if_insufficient_space_for_decompression(output_folder, uncompressed_size)
  File "/var/tmp/packman/packman-common/6.45.1/packman/archive.py", line 104, in raise_if_insufficient_space_for_decompression
    utils.raise_if_less_space(output_path, uncompressed_archive_size * 2)
  File "/var/tmp/packman/packman-common/6.45.1/packman/utils.py", line 507, in raise_if_less_space
    f"Not enough free space available at '{abs_path}' - requested {get_pretty_size(num_bytes)} "
packman.errors.PackmanError: Not enough free space available at '/var/tmp/packman/chk/rtx_plugins/ver-n6hjmb2x' - requested 8.42 GiB  but only 8.42 GiB bytes available!
CMake Error at extern/CMakeLists.txt:46 (message):
  Gathering Nvidia libraries failed


-- Configuring incomplete, errors occurred!
See also "/github/workspace/build/CMakeFiles/CMakeOutput.log".
```
Notably 8.4GiB is larger than the default runner size, so either we have a non-default runner or we're operating in some sort of swap/overhead provided when the runner actually succeeds. If we're not on one of the larger runner sizes, can we sign up for the [larger runner size beta](https://docs.github.com/en/actions/using-github-hosted-runners/using-larger-runners)?

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -17,7 +17,7 @@ jobs:
           - {
               name: 'Windows - MSVC',
               artifact: 'windows-msvc.tar.xz',
-              os: ['self-hosted', 'windows', 'x64', 'vs2022'],
+              os: windows-latest-l,
               cc: 'cl',
               cxx: 'cl',
               build-type: 'Release',
@@ -39,7 +39,7 @@ jobs:
           - {
               name: 'CentOS 7 - GCC',
               artifact: 'linux-gcc.tar.xz',
-              os: ubuntu-22.04,
+              os: ubuntu-latest-m,
               cc: 'gcc-11',
               cxx: 'g++-11',
               build-type: 'Release',
@@ -70,7 +70,7 @@ jobs:
       - name: Conan Set build directory (Windows)
         if: runner.os == 'Windows'
         run: |
-          echo "CONAN_USER_HOME=d:\" >> $env:GITHUB_ENV
+          echo "CONAN_USER_HOME=$env:GITHUB_WORKSPACE" >> $env:GITHUB_ENV
           echo "CONAN_USER_HOME_SHORT=None" >> $env:GITHUB_ENV
 
       # The -v{num} is a cache key buster because we were starting to use the same cache keys from previous tests
@@ -80,7 +80,7 @@ jobs:
         uses: actions/cache@v3
         with:
           path: ${{ env.CONAN_USER_HOME }}/.conan
-          key: conan-${{ matrix.config.name }}-${{ hashFiles('cmake/AddConanDependencies.cmake') }}-v2
+          key: conan-${{ matrix.config.name }}-${{ hashFiles('cmake/AddConanDependencies.cmake') }}-v3
 
       # The -v{num} is a cache key buster to invalidate the Docker image cache
       - name: Load Docker image cache

